### DISCUSSION TITLE
How to specify edit.watch in json config file

### DISCUSSION THREAD

**Discussion by fslef** [twpayne/chezmoi#2653] [Q&A] [answered]
Hi everyone 👋🏻

I have the following json config file:
``` JSON
 {
    "edit": {
        "command": "code",
        "args": "--wait",
        "watch": true
    },
    "git": {
        "autoPush": true
    },
    "data": {
        "github_user": "FSLEF"
    }
}
```

The edit.watch feature is not working, and I am wondering if my configuration file is well formatted.

When typing the `chezmoi data` command, the parameter does not appear.
Here is the command output:

``` JSON
{
  "chezmoi": {
    "arch": "arm64",
    "args": [
      "chezmoi"
      "data"
    ],
    "cacheDir": "/Users/************/.cache/chezmoi",
    "configFile": "/Users/************/.config/chezmoi/chezmoi.json",
    "executable": "/opt/homebrew/bin/chezmoi",
    "fqdnHostname": "*******-MacBook-Pro.local",
    "gid": "20",
    "group": "staff",
    "homeDir": "/Users/************",
    "hostname": "*********-MacBook-Pro",
    "kernel": {},
    "os": "darwin",
    "osRelease": {},
    "sourceDir": "/Users/************/.local/share/chezmoi",
    "uid": "501",
    "username": "************",
    "version": {
      "builtBy": "Homebrew",
      "commit": "*****************************************",
      "date": "2022-12-13T20:48:41Z",
      "version": "2.28.0"
    },
    "windowsVersion": {},
    "workingTree": "/Users/************/.local/share/chezmoi"
  },
  "github_user": "FSLEF"
}
```

Any suggestions ?

**Comment by twpayne** [comment, marked as answer]
> The edit.watch feature is not working, and I am wondering if my configuration file is well formatted.

This was a bug in chezmoi where only the first change to a file was applied. #2654 fixes it and I'll release a new version of chezmoi shortly.

> When typing the `chezmoi data` command, the parameter does not appear.

This is correct. `chezmoi data` shows your template data, whereas `edit.watch` is a configuration variable. To see you chezmoi configuration, use `chezmoi dump-config`.

**Reply by fslef** [reply]
Wonderful thanks a lot. 
Your last big of the year 😅
Thank you and happy new year 🎊🎈🎆
